python-3.10.yaml

```diff
@@ -45,24 +45,16 @@ var-transforms:
 pipeline:
   - uses: git-checkout
     with:
-      expected-commit: 976ea78599d71f22e9c0fefc2dc37c1d9fc835a4
+      expected-commit: ffee63f34445412322a7afc2535731be221cabba
       repository: https://github.com/python/cpython.git
       tag: v${{package.version}}
-      cherry-picks: |
-        # https://github.com/python/cpython/issues/113171
-        3.10/c62c9e518b784fe44432a3f4fc265fb95b651906: CVE-2024-4032
-        3.10/e0264a61119d551658d9445af38323ba94fc16db: CVE-2024-8088
 
   - name: Force use of system libraries
     runs: |
       rm -rf Modules/expat \
         Modules/_ctypes/darwin* \
         Modules/_ctypes/libffi*
 
-  - uses: patch
-    with:
-      patches: CVE-2023-27043.patch
-
   - uses: patch
     with:
       patches: CVE-2023-27043-enable-disable.patch
```

Comment:
Do you need this anymore as the changes are applied in upstream we should not add a way to enable or disable it.

Reply:
@smoser yeah, still need to drop this pipeline.

```
2024/09/11 14:58:37 WARN /bin/sh: can't open CVE-2023-27043-enable-disable.patch: no such file
2024/09/11 14:58:37 ERRO ERROR: failed to build package. the build environment has been preserved:
```